workday/12.1/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -0,0 +1,132 @@
+= Workday - Studio Configuration - Mule 4
+:page-aliases: connectors::workday/workday-studio.adoc, connectors::workday/workday-connector-studio.adoc
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-input-source,Configure an input source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+
+When you run the connector, you can view the app log, as described in <<view-app-log,View the App Log>>.
```

Comment:
Should we add WHY the audience would want or need to do this?
(I know this is templated...but I can't stop from thinking about alternatives.)